spec/decorators/data_request_area_decorator_spec.rb

```diff
@@ -83,12 +83,60 @@
     end
   end
 
+  describe "#request_dates" do
```

Comment:
I think this should be testing where there are multiple data_requests with dates. At the moment it's just retesting the data_request_decorator method